docs/docs-content/enterprise-version/system-management/ssl-certificate-management.md

```diff
@@ -9,17 +9,16 @@ tags: ["palette", "management"]
 keywords: ["self-hosted", "enterprise"]
 ---
 
-When you install Palette, a self-signed certificate is generated and used by default. You can upload your own SSL
-certificate to replace the default certificate.
-
-Palette uses SSL certificates to secure external communication. Internal components communication is by default secured
-and use HTTPS. External communication with Palette, such as the system console, gRPC endpoint, and API endpoint,
-requires you to upload an SSL certificate to enable HTTPS.
+Palette uses Secure Sockets Layer (SSL) certificates to secure internal and external communication with Hypertext
+Transfer Protocol Secure (HTTPS). External Palette endpoints, such as the
```

Comment:
I have a feeling that HTTP/HTTPS are stable acronyms at this point, and we might not want to explicitly clarify them here 🤔